pkg/messaging/v1/invoke_method_response.go

```diff
@@ -115,7 +115,7 @@ func (imr *InvokeMethodResponse) IsHTTPResponse() bool {
 	return imr.r.Status.Code >= 100
 }
 
-// Proto clones the internal InvokeMethodResponse pb object.
```

Comment:
the comment here says it returns a clone. 
Is it necessary to return a clone and not the direct object to make sure no changes are made?

Reply:
We've been going all this time without cloning and it's been working fine :) Cloning would also degrade performance.

Reply:
agreed cloning would degrade performance. as long as in the code path where this Proto is used, we are ok with the direct access, then it's fine.